tests/e2e/specs/saved-cards/saved-cards-my-account.spec.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { CustomerFlow, uiUnblocked } from '../../utils';
+import {
+	fillCardDetails,
+	confirmCardAuthentication,
+} from '../../utils/payments';
+
+const TIMEOUT = 100000;
```

Comment:
Same notes on timeouts as for checkout spec.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/816/commits/b909b29e6c577f4c968a0da0bfa6521ac0b248b9.